src/Compilers/CSharp/Test/Symbol/Symbols/SymbolEqualityTests.cs

```diff
@@ -0,0 +1,418 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Test.Utilities;
+using Xunit;
+using System.Collections.Generic;
+using System;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public class SymbolEqualityTests : CSharpTestBase
```

Comment:
>public [](start = 4, length = 6)

Consider testing `SymbolEqualityComparer.Equals(null, null)`, `Equals(null, other)`, `GetHashCode(null)`.

Reply:
Added Intern_Symbol_equality_with_null test

---
In reply to: [305546931](https://github.com/dotnet/roslyn/pull/37247#discussion_r305546931) [](ancestors = 305546931)